### ISSUE TITLE
TFOptimizer with validation_split doesn

### ISSUE THREAD

**Issue by hkvision** [intel-analytics/analytics-zoo#2271]

**Comment by yangw1234** [comment]
I'll first disable this feature (#2272 ), I think it makes more sense to support it in TFDataset if needed.

**Comment by hkvision** [comment]
It makes sense to support in TFDataset. But to align with Keras API, making this an option in fit is also reasonable.

**Comment by yangw1234** [comment]
> It makes sense to support in TFDataset. But to align with Keras API, making this an option in fit is also reasonable.

yes, it would be ideal if we can make this an option in fit. Do you have any idea to implement it?

**Comment by yangw1234** [comment]
I'll first remove it, we can added it back once we know how to support it.

**Comment by hkvision** [comment]
ok.
